src/pubsub.ts

```diff
@@ -543,30 +543,60 @@ export class PubSub {
   }
   /**
    * Determine the appropriate endpoint to use for API requests, first trying
-   * the local `apiEndpoint` parameter. If the `apiEndpoint` parameter is null
-   * we try Pub/Sub emulator environment variable (PUBSUB_EMULATOR_HOST),
-   * otherwise the default JSON API.
+   * the local `apiEndpoint` parameter. If the `apiEndpoint` parameter is null,
+   * we try the standard `gcloud alpha pubsub` environment variable
+   * (CLOUDSDK_API_ENDPOINT_OVERRIDES_PUBSUB), and failing that, we try the
+   * Pub/Sub emulator environment variable (PUBSUB_EMULATOR_HOST).
+   * Otherwise the default production API is used.
```

Comment:
You probably also want to update https://github.com/googleapis/nodejs-pubsub/blob/a5093c439e5cb13a9d9152d0483373206634ac9b/src/pubsub.ts#L187-L190